src/py42/sdk/__init__.py

```diff
@@ -7,6 +9,8 @@
 from py42.services._connection import Connection
 from py42.usercontext import UserContext
 
+warnings.simplefilter("always", DeprecationWarning)
```

Comment:
DeprecationWarnings are muted by default in python (not sure why). Do we want them to always be sent?